internal/common/config/configer.go

```diff
@@ -130,13 +207,43 @@ var processEnvAgent = func(config *AgentConfig) error {
 	return nil
 }
 
+func setupDefaultServerValues(config *ServerConfig,
+	defaultAddress string,
+	defaultStoreInterval int64,
+	defaultFileStoragePath string,
+	defaultRestore bool) {
+	if config.Address == unknownStringFieldValue {
+		config.Address = defaultAddress
+	}
+	if config.StoreInterval == unknownIntFieldValue {
+		config.StoreInterval = defaultStoreInterval
+	}
+	if config.FileStoragePath == unknownStringFieldValue {
+		config.FileStoragePath = defaultFileStoragePath
+	}
+	if !config.hasRestore {
+		config.Restore = defaultRestore
+	}
+}
+
 func setupDefaultAgentValues(config *AgentConfig,
+	defaultAddress string,
 	defaultRepInterval time.Duration,
 	defaultPollInterval time.Duration) {
+	if config.Address == "" {
+		config.Address = defaultAddress
+	}
 	if config.ReportInterval == 0 {
 		config.ReportInterval = int64(defaultRepInterval)
 	}
 	if config.PollInterval == 0 {
 		config.PollInterval = int64(defaultPollInterval)
 	}
 }
+
+func (cfg ServerConfig) String() string {
```

Comment:
Please use efficient formatting of strings. Please check every package!

Reply:
done